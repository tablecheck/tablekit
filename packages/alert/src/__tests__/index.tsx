import { mount } from '@tablecheck/tablekit-enzyme';

import { Alert } from '../index';

beforeEach(() => {
  global.console.error = jest.fn();
});

describe('Alert', () => {
  test('should be possible to create a component', () => {
    const wrapper = mount(<Alert />);
    expect(wrapper).not.toBeUndefined();
  });

  test('should correctly apply styling', () => {
    expect(mount(<Alert />)).toMatchSnapshot();
  });

  test('should allow passing of custom className', () => {
    expect(mount(<Alert className="test-class" />)).toMatchSnapshot();
  });
});
