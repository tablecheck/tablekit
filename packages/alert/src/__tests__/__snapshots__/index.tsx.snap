// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Alert should allow passing of custom className 1`] = `
<Alert
  className="test-class"
/>
`;

exports[`Alert should correctly apply styling 1`] = `<Alert />`;
